com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/model/ConfigurationSettingsStorable.java

```diff
@@ -25,10 +25,13 @@
  */
 @Value
 public class ConfigurationSettingsStorable {
-  private Object profiles;
   private List<String> paths;
 
+  public ConfigurationSettingsStorable(List<String> paths) {
+    this.paths = paths;
+  }
+
   public static ConfigurationSettingsStorable emptyConfigurationSettingsStorable() {
```

Comment:
This method is unused

Reply:
It was intended to be used when we cannot parse the settings. Did the idea change?

Reply:
@temanbrcom is right, it is for testing purpose or for the moment we will retrieve a wider configuration section

Reply:
I still see that is not used, is planned to be reused later?

Reply:
yes, if needed

Reply:
Ok so If I understood correctly at moment we don't need it and we can remove it - we can restore it back from the version control if needed, agree?

Reply:
sure, done